src/Apps/Artist/ArtistApp.tsx

```diff
@@ -49,33 +45,17 @@ export class ArtistApp extends React.Component<ArtistAppProps> {
           <Row>
             <Col>
               <NavigationTabs artist={artist} />
-
               <Spacer mb={3} />
 
-              {/*
-              When clicking nav links, wait for fetch to complete before
-              transitioning to new route
-            */}
-
-              <Subscribe to={[PreloadLinkState]}>
-                {({ state: { isLoading } }: PreloadLinkState) => {
-                  return (
-                    <LoadingArea isLoading={isLoading}>{children}</LoadingArea>
-                  )
-                }}
-              </Subscribe>
+              <Box minHeight="30vh">{children}</Box>
             </Col>
           </Row>
 
-          {typeof window !== "undefined" && (
-            <LazyLoadComponent threshold={1000}>
-              <Row>
-                <Col>
-                  <RecentlyViewed />
-                </Col>
-              </Row>
-            </LazyLoadComponent>
-          )}
+          <Row>
+            <Col>
+              <RecentlyViewed />
+            </Col>
+          </Row>
```

Comment:
Removed lazy loading on this; we were seeing double preloaders as this is loading in. Also, it would frequently cause our layouts to jump as we scroll. Not sure if it's necessary -- user is logged in, and it's behind a query renderer.

Reply:
Are we still using the dep elsewhere?

Reply:
It is; on other pages without tabbed navigation it's still in place.